code/modules/mob/living/carbon/human/ai/action_datums/take_cover.dm

```diff
@@ -6,7 +6,8 @@
 	if(!brain.current_cover)
 		return 0
 
-	if(brain.in_cover && !(get_dist(brain.tied_human, brain.current_target) > brain?.gun_data?.minimum_range))
+	var/cover_distance_check = (get_dist(brain.tied_human, brain.current_target) <= brain?.gun_data?.minimum_range)
+	if(brain.in_cover && (!brain.current_target || !brain.gun_data || cover_distance_check))
```

Comment:
why make this a var

Reply:
the line was so long it couldn't fit in my vsc screen space
again, readability is subjective, so i can just put it out and make a one giant if() check if you will